json-rpc/spec.json

```diff
@@ -196,6 +196,94 @@
 				"$ref": "#/components/contentDescriptors/GasPrice"
 			}
 		},
+		{
+			"name": "eth_feeHistory",
+			"summary": "Transaction fee history",
+			"description": "Returns transaction base fee and effective priority fee history for the requested/supported block range.",
```

Comment:
What would the behaviour be before EIP-1559 blocks? I think it would be good to mention that upfront in the description.

Reply:
Also, these should be listed "base fee per gas" and "effective priority fee per gas".